### ISSUE TITLE
RimThreaded.Building_Door_Patch.get_DoorPowerOn - NullReferenceException

### ISSUE THREAD

**Issue by Mythos** [cseelhoff/RimThreaded#56]
I've loaded my savegame and let the game run for just a few seconds on Speed 2-4.

```
Root level exception in OnGUI(): System.NullReferenceException: Object reference not set to an instance of an object
  at RimThreaded.Building_Door_Patch.get_DoorPowerOn (RimWorld.Building_Door __instance, System.Boolean& __result) [0x00011] in <a4fd077b79864841aff3f060c1535258>:0 
  at (wrapper dynamic-method) RimWorld.Building_Door.RimWorld.Building_Door.get_DoorPowerOn_Patch1(RimWorld.Building_Door)
  at RimWorld.Building_Door.get_SlowsPawns () [0x00000] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.AI.Pawn_PathFollower.NextCellDoorToWaitForOrManuallyOpen () [0x0001f] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.PawnTweener.MovedPercent () [0x00048] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.PawnTweener.TweenedPosRoot () [0x00021] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.PawnTweener.ResetTweenedPosToRoot () [0x00000] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.PawnTweener.PreDrawPosCalculation () [0x0001d] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.Pawn_DrawTracker.get_DrawPos () [0x00000] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.Pawn.get_DrawPos () [0x00006] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at Verse.GenUI.ThingsUnderMouse (UnityEngine.Vector3 clickPos, System.Single pawnWideClickRadius, RimWorld.TargetingParameters clickParams) [0x00036] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at RimWorld.Selector+<SelectableObjectsUnderMouse>d__39.MoveNext () [0x0009e] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at System.Collections.Generic.List`1[T]..ctor (System.Collections.Generic.IEnumerable`1[T] collection) [0x00077] in <567df3e0919241ba98db88bec4c6696f>:0 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00018] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at RimWorld.Selector.SelectUnderMouse () [0x0004c] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at (wrapper dynamic-method) RimWorld.Selector.RimWorld.Selector.HandleMapClicks_Patch2(RimWorld.Selector)
  at RimWorld.Selector.SelectorOnGUI () [0x00000] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at RimWorld.MapInterface.HandleLowPriorityInput () [0x0000f] in <7b345446a85d4ed599f9c604ae61854b>:0 
  at (wrapper dynamic-method) RimWorld.UIRoot_Play.RimWorld.UIRoot_Play.UIRootOnGUI_Patch1(RimWorld.UIRoot_Play)
  at (wrapper dynamic-method) Verse.Root.Verse.Root.OnGUI_Patch1(Verse.Root)
Verse.Log:Error(String, Boolean)
Verse.Root:Verse.Root.OnGUI_Patch1(Root)

```

**Comment by Mythos** [comment]
I've exported my mod list via RimPy just in case it's useful to you. :-)

[Mods.txt](https://github.com/cseelhoff/RimThreaded/files/5210604/Mods.txt)

(btw big thanks for trying to tackle this long time issue!)

**Comment by cseelhoff** [comment]
Thank you for your bug submission! This bug in particular has been haunting me. I suspect a mod conflict. We will see! Do you have a consistent way to reproduce this?

**Comment by Mythos** [comment]
I just started the game and first thing I did was disable all robots (from Misc. Robots as they are "pawns").
Then I let the game run on Speed 3/4 and got a ton of timeout errors.
So I thought I'll draft my colonists and then I got a ton of those errors. According to the stack trace it's related to Dub's Minimap.

![image](https://user-images.githubusercontent.com/416568/92989248-40c2b380-f4d3-11ea-917a-0aec619ccf3a.png)
